chain-crypto/src/key.rs

```diff
@@ -167,6 +167,12 @@ impl<A: AsymmetricKey> SecretKey<A> {
     pub fn from_binary(data: &[u8]) -> Result<Self, SecretKeyError> {
         Ok(SecretKey(<A as AsymmetricKey>::secret_from_binary(data)?))
     }
+    // get ownership of the secret key, allowing to get its byte representation.
```

Comment:
Could be a doc comment, but then again the whole module API is currently undocumented.